README.md

```diff
@@ -2,8 +2,14 @@
 General Language Independent Driver for the Enterprise (GLIDE) for Redis, is an AWS-sponsored, open-source Redis client. GLIDE for Redis works with any Redis distribution that adheres to the Redis Serialization Protocol (RESP) specification, including open-source Redis, Amazon ElastiCache for Redis, and Amazon MemoryDB for Redis.
 Strategic, mission-critical Redis-based applications have requirements for security, optimized performance, minimal downtime, and observability.  GLIDE for Redis is designed to provide a client experience that helps meet these objectives. It is sponsored and supported by AWS, and comes pre-configured with best practices learned from over a decade of operating Redis-compatible services used by hundreds of thousands of customers. To help ensure consistency in development and operations, GLIDE for Redis is implemented using a core driver framework, written in Rust, with extensions made available for each supported programming language. This design ensures that updates easily propagate to each language and reduces overall complexity. In this Preview release, GLIDE for Redis is available for Python and Javascript (Node.js), with support for Java actively under development.
 
-## Supported Redis Versions
-GLIDE for Redis is API-compatible with open source Redis version 6 and 7.
+## Supported Engine Versions
+GLIDE for Redis is API-compatible with the following engine versions:
+
+| Engine Type        | Version |
+|--------------------|---------|
```

Comment:
I would add ValKey 7.2.5, its show's something better, we keep updating with the last releases of ValKey, and users can trust us to keep up with releases

Reply:
Valkey 7.2 means that we support all minor versions of valkey 7.2, including 7.2.5. I think adding 7.2.5 can be confusing